lib/assessment.js

```diff
@@ -4,24 +4,37 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+var min = function min(num1,num2) {
+  return Math.min(num1,num2);
```

Comment:
Because we don't know how many arguments our function will be given, we have to use the array-like object `arguments` to keep track of whatever arguments we get. 

Additionally, the idea was more that you would create the functionality of `Math.min` from scratch. Can you think about how you might do that? 
(HINT: It's doable with a single `for` loop.)